airbyte-integrations/connectors/source-hubspot/metadata.yaml

```diff
@@ -23,6 +23,13 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      1.10.0:
+        message: >-
+          This version changes PropertyHistory stream to 3 different streams as we now support not only Contacts but also Companies and Deals to fetch their property history.
```

Comment:
This version eliminates the PropertyHistory stream in favor of creating 3 different streams, Contacts, Companies, and Deals, which can now all fetch their property history.
It will affect only users who use PropertyHistory stream, who will need to fix schema conflicts in replication tab and sync ContactsPropertyHistory stream instead of PropertyHistory.

Reply:
Done